tests/env/typescript-browser/package.json

```diff
@@ -13,8 +15,10 @@
   "author": "Charlotte Vermandel",
   "license": "ISC",
   "devDependencies": {
-    "webpack": "^4.43.0",
+    "@webpack-cli/serve": "^1.6.1",
```

Comment:
You need both `@webpack-cli/serve` and `webpack-dev-server` ?

Reply:
Yes! It does not work without both